### PR TITLE
clash.net: Add version 1.0.1

### PR DESCRIPTION
**Pull request by lifansama**
A Clash GUI Proxy For Windows Based On .NET 5

> scoop install clash.net
Installing 'clash.net' (1.0.1) [64bit]
Loading Clash.NET.1.0.1.x64.7z from cache
Checking hash of Clash.NET.1.0.1.x64.7z ... ok.
Extracting Clash.NET.1.0.1.x64.7z ... done.
Linking ~\scoop\apps\clash.net\current => ~\scoop\apps\clash.net\1.0.1
Creating shortcut for Clash.NET (ClashDotNet.exe)
'clash.net' (1.0.1) was installed successfully!

### CODE REVIEW

File: bucket/clash.net.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "homepage": "https://github.com/ClashDotNetFramework/ClashDotNetFramework",
+    "description": "A Clash GUI Proxy For Windows Based On .NET 5",
+    "version": "1.0.1",
+    "license": "none",
```
Comment (Ash258) [inline bucket/clash.net.json:5]:
```suggestion
    "license": "Unknown",
```

File: bucket/clash.net.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+    "homepage": "https://github.com/ClashDotNetFramework/ClashDotNetFramework",
+    "description": "A Clash GUI Proxy For Windows Based On .NET 5",
```
Comment (Ash258) [inline bucket/clash.net.json:3]: Keeping `For Windows` and dotnet version is not needed. 

Scoop is only for windows and version of dotnet is not relevant

File: bucket/clash.net.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "version": "1.0.1",
+    "license": "none",
+    "architecture": {
```
Comment (Ash258) [inline bucket/clash.net.json:6]: Add suggest for `windowsdesktop-runtime`

**Review by Ash258 — CHANGES_REQUESTED** [review]
- Add autoupdate with hash extraction from `checksum` file on releases page
- Enable portable mode and add persist for all relevant files/directories

image

**Comment by issaclin32** [comment]
The software is discontinued and removed by the developer.
https://github.com/ClashDotNetFramework/ClashDotNetFramework

**Pull request closed without merging**